battery_inverters/sma_sunny_island/README.md

```diff
@@ -1,12 +1,13 @@
 # SMA Sunny Island
 
-This _Enapter Device Blueprint_ integrates **SMA Sunny Island** battery inverter via [Modbus TCP](https://developers.enapter.com/docs/reference/vucm/modbustcp) implemented on [Enapter Virtual UCM](https://handbook.enapter.com/software/software.html#%F0%9F%92%8E-virtual-ucm).
+This [Enapter Device Blueprint](https://github.com/Enapter/wip-marketplace#blue_book-enapter-device-blueprints) integrates **SMA Sunny Island** battery inverter via [Modbus TCP](https://developers.enapter.com/docs/reference/vucm/modbustcp) implemented on [Enapter Virtual UCM](https://handbook.enapter.com/software/software.html#%F0%9F%92%8E-virtual-ucm).
 
-Use [Enapter Gateway](https://handbook.enapter.com/software/gateway/2.0.0/setup/) to run Virtual UCM.
+## Connect to Enapter
 
-## Setup and Connection
-
-You will need the following information about your SMA Sunny Island device:
-
-- Modbus IP address (use either static IP or DHCP reservation);
-- Modbus Unit ID (can be found in SMA Web interface).
+- Sign up to Enapter Cloud using [Web](https://cloud.enapter.com/) or mobile app ([iOS](https://apps.apple.com/app/id1388329910), [Android](https://play.google.com/store/apps/details?id=com.enapter&hl=en)).
+- Use [Enapter Gateway](https://handbook.enapter.com/software/gateway/2.0.0/setup/) to run Virtual UCM.
+- Create [Enapter Virtual UCM](https://handbook.enapter.com/software/software.html#%F0%9F%92%8E-virtual-ucm).
+- [Upload](https://developers.enapter.com/docs/tutorial/uploading-blueprint/) this blueprint to Enapter Virtual UCM.
+- Use `Configure` command in Enapter mobile app or Web to set up SMA Sunny Island communication parameters:
+  - _Modbus IP address_, use either static IP or DHCP reservation. Check your network router manual for configuration instructions.
+  - _Modbus Unit ID_, can be found in SMA Web interface. Find out [how to configure it](fixme) in SMA manual.
```

Comment:
@nikitug could'nt find relevant link, only these:
https://files.sma.de/downloads/EDMx-Modbus-TI-en-15.pdf - info about Modbus Unit ID
https://files.sma.de/downloads/SI44M-80H-13-BE-en-13.pdf - use of Web interface (but without changing unit ID)

Reply:
Ok, let's add that to the reference section anyway